projects/map-cesium/src/lib/map-cesium.service.ts

```diff
@@ -224,38 +224,41 @@ export class MapCesiumService {
   }
 
   // Set initial oblique view, see https://cesium.com/learn/cesiumjs/ref-doc/Camera.html
+  //subtract 90°, to get the same behavior as in openlayers
   public setViewAngle(viewAngle: number) {
```

Comment:
Here, I would allow the flyTo options to be passed and merged. I implemented this in one of our projects so that users don't have to set a timeout and can listen to flyTo.complete instead. 

```typescript
/**
*  Set initial oblique view, see https://cesium.com/learn/cesiumjs/ref-doc/Camera.html
*  subtract 90°, to get the same behavior as in openlayers
*  options of viewer.camera.flyTo`
*/
public setViewAngle(viewAngle: number, options?: any`) {
  const flyToOptions = Object.assign({
      destination: this.viewer.camera.position,
      orientation: {
         heading: this.viewer.camera.heading,
         pitch: CesiumMath.toRadians(viewAngle-90),
         roll: this.viewer.camera.roll,
    });
    }, options || {});
    this.viewer.camera.flyTo(flyToOptions)
}
```